contracts/interfaces/V3SpokePoolInterface.sol

```diff
@@ -91,6 +91,22 @@ interface V3SpokePoolInterface {
         uint256 updatedOutputAmount;
         FillType fillType;
     }
+    // Represents the parameters required for a V3 deposit operation in the SpokePool.
+    struct DepositV3Params {
+        bytes32 depositor;
+        bytes32 recipient;
+        bytes32 inputToken;
```

Comment:
See [this comment](https://github.com/across-protocol/contracts/pull/766/files#r1855152491)